src/unity/paths/operator_accounts_accountId_users_userId.yml

```diff
@@ -1,9 +1,9 @@
 delete:
-  operationId: RemoveUserFromAccount
+  operationId: removeUserFromAccount
```

Comment:
From what I've seen, it looks like operation IDs are typically UppercaseFirst, but I see that a number of them are changed to lowercaseFirst here. Which one should it be?

Reply:
im not sure if was a standard, i can look at other contracts and adjust.